doc/build/dts/api-usage.rst

```diff
@@ -50,6 +50,8 @@ By alias
    :zephyr:code-sample:`blinky`, which uses the ``led0`` alias) that need to
    refer to *some* device of a particular type ("the board's user LED") but
    don't care which one is used.
+   You may also use :c:macro:`DT_HAS_ALIAS()` to verify whether an alias
```

Comment:
guess we need to wait for the conclusion on this one now: https://github.com/zephyrproject-rtos/zephyr/pull/80992 before being able to approve this PR

Reply:
> Nit/not directly related to this PR since it's wrongly set elsewhere but it should be :c:macro: not :c:func:

The commit regarding the dts documentation stems from this PR. I look forward to discussing the result.

Reply:
yes, I know.

But this PR should be ensured that syntax used in the rst is aligned with rest of documentation, and syntax for rest of documentation was being decided in #80992 :wink: